packages/webui/src/client/ui/SegmentTimeline/SegmentContextMenu.tsx

```diff
@@ -145,6 +165,18 @@ export const SegmentContextMenu = withTranslation()(
 										)}
 									</>
 								)}
+
+								{RenderUserEditOperations(
+									isPartEditAble,
+									part.instance.rundownId,
+									part.instance.part.title,
+									part.instance.part.userEditOperations,
+									{
+										segmentExternalId: segment?.externalId,
+										partExternalId: part.instance.part.externalId,
+										pieceExternalId: undefined,
+									}
```

Comment:
I feel that it's a little bit icky that the externalId's are being passed around in the GUI. Maybe this could pass regular SegmentIds, PartIds, and PieceIds? and have `RenderUserEditOperations` resolve them to a set of information that is being used by the backend?